sycl/test/basic_tests/known_identity.cpp

```diff
@@ -1,4 +1,4 @@
-// RUN: %clangxx -fsycl -Xclang -verify %s -Xclang -verify-ignore-unexpected=note,warning -o %t.out -std=c++17
+// RUN: %clangxx %fsycl-host-only -Xclang -verify %s -Xclang -verify-ignore-unexpected=note,warning -o %t.out -std=c++17 %sycl_lib -L%sycl_libs_dir
```

Comment:
Though this is clever, I worry that having us do explicit linking here is fragile. Could the test potentially be split into a compile-only test and a test that we will need to run?

Reply:
yes, it could be. We have runtime time asserts there that could be moved to unit test as example. Although it looks more consistent to have complete test.

Reply:
Discussed offline. This seems like the better option given the causes of slow compilation.